src/client/pages/registration/components/Organisms/RegistrationForm.jsx

```diff
@@ -0,0 +1,19 @@
+import { RegistrationInputs } from '../Molecules/RegistrationInputs.jsx';
+import { RectangleButton } from 'client/components/Atoms/RectangleButton/RectangleButton.jsx';
+import { RegistrationLink } from '../Atoms/RegistrationLink.jsx';
+import css from './RegistrationForm.module.css';
```

Comment:
Create imports in our `package.json` and use it like other path.

f.ex. 
`"imports": {`
`    "#controllers/*": "./controllers/*"`
`    }`